BlueMap/Chat/src/main/java/codes/antti/auth/bluemap/chat/BlueMapChatPlugin.java

```diff
@@ -67,12 +97,12 @@ public void run() {
 			}
 		}, 0, 30, TimeUnit.SECONDS);
 
-		getServer().getPluginManager().registerEvents(this, this);
+		var plugin_manager = getServer().getPluginManager();
+		plugin_manager.registerEvents(this, this);
```

Comment:
Why was this put in a variable? I don't see it being used anywhere.

Reply:
I think it was because just in case if it were used in the future. Will fix this